README.md

```diff
@@ -204,25 +253,141 @@ This endpoint returns a list of all existing criteria. It will return an empty a
+| active              | Boolean  | *Optional*. Passing in `false` will create a deactivated link. Links cannot be reactivated once they have been deactivated. |
+
+### Scores
+
+A score represents whether a state meets a certain criteria.
+
+| Name            | Type    | Notes                   |
+|-----------------|---------|-------------------------|
+| id              | Integer | Primary key             |
+| state_code      | String  | Foreign key             |
+| criterion_id    | Integer | Foreign key             |
+| meets_criterion | Boolean |                         |
+
+#### POST /scores
+
+This endpoint creates a new score. Note that scores CANNOT be updated, only overwritten. To overwrite
+a score, create a new score for the same state and criterion.
+
+This endpoint requires [authentication](#Authentication). It accepts a JSON body with the following format:
+
+| Name            | Type    | Notes                                                     |
+|-----------------|---------|-----------------------------------------------------------|
+| state_code      | String  | *Required*. The state code to which the score is related. |
+| criterion_id    | Integer | *Required*. The criterion to which the score is related.  |
+| meets_criterion | Boolean | *Required*. Whether the state meets the criterion.        |
+
 ### Grades
 
 A **state grade** represents the overall grade assigned to a state based on their survivor wealth policies.
 
 |    Name    |  Type   |          Notes          |
 |------------|---------|-------------------------|
 | id         | Integer | Primary key             |
-| state_code | String  |                         |
+| state_code | String  | Foreign key             |
 | grade      | Integer | One of (-1, 0, 1, 2, 3) |
 
 A **state category grade** represents the grade assigned to a state based on a specific category.
 
 |    Name     |  Type   |          Notes          |
 |-------------|---------|-------------------------|
 | id          | Integer | Primary key             |
-| state_code  | String  |                         |
-| category_id | Integer |                         |
+| state_code  | String  | Foreign key             |
+| category_id | Integer | Foreign key             |
 | grade       | Integer | One of (-1, 0, 1, 2, 3) |
 
 #### GET /grades/{code}
 
 Returns the state's overall grade, and its grades for each category. If no state with that code exists, it will return a 404 response code.
+
+#### POST /grades/{code} (UPCOMING)
+
+This endpoint creates an overall state grade. Note that state grades CANNOT be updated, only overwritten. To overwrite a grade, create a new grade for the same state.
+
+This endpoint requires [authentication](#Authentication). It accepts a JSON body with the following format:
+
+|    Name    |  Type   |          Notes          |
+|------------|---------|-------------------------|
+| grade      | Integer | *Required*. One of (-1, 0, 1, 2, 3) |
```

Comment:
The POST body only needs the grade because the state id comes from the query params.